public/app/percona/shared/components/PerconaBootstrapper/PerconaNewVersion/PerconaNewVersion.constants.ts

```diff
@@ -0,0 +1,11 @@
+export const Messages = {
```

Comment:
We usually have a separate `PerconaNewVersion.messages.ts` file just for the strings.